official/benchmark/models/resnet_imagenet_test.py

```diff
@@ -26,12 +26,13 @@
 from official.utils.testing import integration
 from official.vision.image_classification.resnet import imagenet_preprocessing
 
-
+# TBC: joint clustering and tuning is not supported yet so only one flag should be selected
```

Comment:
Could you clarify what this means?

Reply:
It was meant to be `clustering and pruning`.
I think when the pruning and clustering flag both enabled there was a problem during testing. Perhaps we could just delete this comment?

Reply:
Resolving since you deleted the comment.